website/docs/examples/controller_header/README.md

```diff
@@ -0,0 +1,165 @@
+---
+id: controller-info
+title: Controller 101
+---
+
+# Controller
+
+The Controller is the autonomy hub of the Amiga.
+It takes in user commands and turns them into actions the robot can perform.
+
+The main goal of the Controller is to act as a bridge between what you want the Amiga to
+do and how the robot actually does it. It hides the complicated details, letting users focus
+on their main tasks without needing to know the complex workings underneath.
+
+In short, the Controller does the heavy lifting, allowing users to easily command the Amiga without
+deep technical knowledge.
+
+## Subscriptions
+
+The `Controller` is a client of the following services:
+
+- Canbus
+- Filter (state estimation)
+
+The State estimation filter service is a client of the following services:
+
+- Canbus
+- GPS
+- Oak0
+
+For this reason, all of the above-mentioned services must be up and running for the `Controller`
+to work.
+
+## Requirements
+
+There are a few requirements on the `/state` output of the state estimation filter
+for the controller service to consider the results valid and allow for following a track.
+These include:
+
+- GPS service is connected to an RTK base station
+- State estimation is receiving all required sensor data
+- State estimation results have converged
+  - Requires a few seconds of driving around after starting the filter service
+
+## API
+
+- `/append_track`: Add a single waypoint to the end of the current track.
+- `/cancel`: Terminate any ongoing operations and bring the Amiga to a stop.
+- `/extend_track`: Add additional waypoints to the current track.
+- `/follow_track`: Instruct the Amiga to commence following the previously set track.
+- `/get_pose`: Retrieve the current position and orientation of the Amiga.
+- `/get_track_end`: Returns the final waypoint of the currently set track.
+- `/replace_track`: Provide a new track for the Amiga to follow, replacing the current one.
+- `/set_track`: Provide a specific track (series of waypoints) for the Amiga to follow.
+- `/stop`: Instruct the Amiga to halt all movements immediately.
```

Comment:
Per @Hackerman342's API proposal, let's document only:
* `/get_pose`
* `/set_track`
* `/follow_track`
* `/stop`

Since that's our recommended flow and shouldn't change drastically. The rest may change more significantly, if we expose them at all.

Reply:
#214